### PR TITLE
Companion for substrate#10632

### PR DESCRIPTION
**Pull request by koushiro**
Companion for paritytech/substrate#10632

### CODE REVIEW

File: polkadot-parachains/src/cli.rs [hunk 1]
```diff
@@ -52,69 +52,70 @@ pub enum Subcommand {
...
 	/// The custom benchmark subcommmand benchmarking runtime pallets.
-	#[structopt(name = "benchmark", about = "Benchmark runtime pallets.")]
+	#[clap(name = "benchmark", about = "Benchmark runtime pallets.")]
 	Benchmark(frame_benchmarking_cli::BenchmarkCmd),
 
 	/// Try some testing command against a specified runtime state.
 	TryRuntime(try_runtime_cli::TryRuntimeCmd),
 
 	/// Key management CLI utilities
+	#[clap(subcommand)]
```
Comment (bkchr) [inline polkadot-parachains/src/cli.rs:62]: Still don't get why we need this here and not for the others.

Comment (bkchr) [reply polkadot-parachains/src/cli.rs:62]: What is the error when we don't do this? Does it fail at compile time?

Comment (koushiro) [reply polkadot-parachains/src/cli.rs:62]: yeah, it will be fail at compile time

**Review by bkchr — APPROVED** [review]

**Comment by KiChjang** [comment]
bot merge

**Pull request merged**